themes/default/content/docs/guides/pulumi-packages/how-to-author.md

```diff
@@ -93,6 +98,16 @@ Once you've authored and tested your package locally, you can publish it to make
 
 Future iterations of this guide will cover how to publish the first three items; for now, these are left as an exercise for the package author.
 
+The URL used to download a plugin is derived from `pluginDownloadURL`, as specified in the schema.[^1] Pulumi expects to find a plugin at
```

Comment:
```suggestion
The URL used to download a plugin is derived from `pluginDownloadURL`, as specified in the schema.[^1] Pulumi expects to find a plugin at

```

Reply:
☝️ 👇 These two suggestions should appease the linter. 😄  (`make lint` for future reference should surface these locally.)